deployment/docker/rules/alerts.yml

```diff
@@ -119,15 +119,3 @@ groups:
           description: "High rate of slow inserts on \"{{ $labels.instance }}\" may be a sign of resource exhaustion
             for the current load. It is likely more RAM is needed for optimal handling of the current number of active time series.
             See also https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3976#issuecomment-1476883183"
-
-      - alert: LabelsLimitExceededOnIngestion
```

Comment:
I think it makes sense to move `RowsRejectedOnIngestion` alert from cluster/single rules file to `alerts-health` file. Don't forget to rm `dashboard` annotation and update description to remove `VM` mentioning - it should be abstract which component is rejecting data.